projects/schematics/src/migrations/5_0/rename-symbol/rename-symbol.ts

```diff
@@ -0,0 +1,11 @@
+import { Rule, Tree } from '@angular-devkit/schematics';
+import { RenamedSymbol } from '../../../shared/utils/file-utils';
+import { migrateRenamedSymbols } from '../../mechanism/rename-symbol/rename-symbol';
+
+export const RENAMED_SYMBOLS_DATA: RenamedSymbol[] = [];
+
+export function migrate(): Rule {
+  return (tree: Tree) => {
+    return migrateRenamedSymbols(tree, [...RENAMED_SYMBOLS_DATA]);
```

Comment:
Do we have to create a new array here? 🧐🤔
If you copy-pasted it from v4 (or v3?), let's fix it there as well.